### DISCUSSION TITLE
Revert Explore tab

### DISCUSSION THREAD

**Discussion by Codel1417** [community/community#18723] [Mobile]
The explore tab used to just have trending and recommended repos but now is cluttered "personalized" crap

**Comment by eliperkins** [comment]
Hey @Codel1417, thanks for your feedback here.

Trending repos are still available in the new Explore tab, by tapping "Trending Repositories" at the top the Explore view:

![Simulator Screen Shot - iPhone Xʀ - 2022-06-17 at 10 21 20](https://user-images.githubusercontent.com/1051453/174317360-47e94cd6-d611-48c8-8ccf-a9e08ccca8cf.png)

Recommended repos are what follows in the Explore view just below that, showing activity from users you follow.

If there are specific things that you'd like to see in the Explore tab, we'd love to hear what exactly you want to see! Thanks for your input.

**Reply by Codel1417** [reply]
The recommended repos are mixed in with a bunch of useless clutter though

**Reply by eliperkins** [reply]
Got it! So what I'm hearing from you is that you only want to see recommended repos in your Explore feed with no other content like followed users or new releases. Is that correct?

**Reply by Codel1417** [reply]
Yes. Currently the feed has no filtering controls

**Reply by eliperkins** [reply]
Definitely. I hear you. On GitHub.com, the feed has a set of filters that allow folks to see exactly the content that's important to them. Would something like this be helpful to you on GitHub Mobile as well?

<img width="376" alt="Screen Shot 2022-06-17 at 10 35 06 AM" src="https://user-images.githubusercontent.com/1051453/174319582-4ffa0933-958c-4c06-96d0-6a8f4e5acd99.png">

**Reply by akamud** [reply]
Yes please bring the filters to mobile too.

**Comment by Codel1417** [comment]
Yes but on the website filtering out everything else still has 2 user follow suggestions for every repo suggestion. Can't easily return to just repo suggestions

**Comment by Alas1980** [comment]
Alguien sabe como revisar remotamente la actividad de un iphone
